packages/common/src/management/management.helper.ts

```diff
@@ -1,11 +1,28 @@
 namespace Online {
 
-  export function getManagementJolokiaPath(pod: any, port: number): string {
+  export type Pod = {
+    metadata: {
+      namespace: string;
+      name: string;
+      annotations?: {
+        [key: string]: string;
+      };
+    };
+  };
+
+  export function getManagementJolokiaPath(pod: Pod, port: number): string {
     const namespace = pod.metadata.namespace;
     const name = pod.metadata.name;
-    const protocol = pod.metadata.annotations['hawt.io/protocol'] || 'https';
-    const jolokiaPath = pod.metadata.annotations['hawt.io/jolokiaPath'] || '/jolokia';
+    const protocol = getAnnotation(pod, 'hawt.io/protocol', 'https');
+    const jolokiaPath = getAnnotation(pod, 'hawt.io/jolokiaPath', '/jolokia');
     return `/management/namespaces/${namespace}/pods/${protocol}:${name}:${port}${jolokiaPath}`;
   }
 
+  function getAnnotation(pod: Pod, name: string, defaultValue: string): string {
+    if (pod.metadata.annotations) {
+      return pod.metadata.annotations[name];
```

Comment:
It should return `defaultValue` in case the annotation value is undefined.

Reply:
Thanks. Fixed.